fedora-toolbox

```diff
@@ -246,6 +250,66 @@ enter()
 )
 
 
+list_images()
+(
+    output=$($prefix_sudo podman images \
+            --filter "label=com.redhat.component=fedora-toolbox" \
+            --format "{{.ID}} {{.Repository}}:{{.Tag}} {{.Created}}" \
+            | sed 's/ \{2,\}/\t/g' | column --separator $'\t' --table \
+            --table-columns "IMAGE ID,IMAGE NAME,CREATED")
+    if ! [[ $output = "" ]]; then
```

Comment:
Would be better to use single brackets and put quotes around `$output` instead for POSIX correctness.

Reply:
Changed.